conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultMessage.kt

```diff
@@ -3,54 +3,29 @@ package com.hodamohammadi.conveyor.models
 import com.stfalcon.chatkit.commons.models.IMessage
 import com.stfalcon.chatkit.commons.models.IUser
 import java.io.Serializable
-import java.util.*
+import java.util.Date
 
 /**
  * Default model for a single message.
  */
-class DefaultMessage: IMessage, Serializable {
-    private var id: String
-    private var text: String
-    private var createdAt: Date
-    private var user: IUser
-
-    constructor(id: String, text: String, createdAt: Date, user: DefaultUser) {
-        this.id = id
-        this.text = text
-        this.createdAt = createdAt
-        this.user = user
-    }
-
-    fun setId(id: String) {
-        this.id = id
-    }
+data class DefaultMessage(val messageId: String,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/itsAlexNguyen/Conveyor/blob/e9a571e2f03220124bea1aedfe5c6054655da1e0/conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultMessage.kt#L11"></span>Parameter should be on a separate line (unless all parameters can fit a single line)</td>
    </tr>
  </tbody>
</table>